app/Services/PostService.php

```
@@ -48,4 +39,17 @@ public function create($data, $signupId, $transactionId)
 
         return $post;
     }
+
+    protected function resolvePostRepository($type)
+    {
+        switch ($type) {
+            case 'post_photo':
+                $this->repository = app('Rogue\Repositories\PhotoRepository');
+
+                break;
+
+            default:
+                break;
```

This should probably throw an exception.